src/test/Components/CreateImageWizard/CreateImageWizard.content.test.js

```diff
@@ -868,3 +873,53 @@ describe('Step Custom repositories', () => {
     await waitFor(() => expect(rows).toHaveLength(10));
   });
 });
+
+describe('On Recreate', () => {
+  const user = userEvent.setup();
+  const setUp = async () => {
+    jest.mock('../../../store/index.js');
+
+    const state = mockStateRecreateImage;
+
+    store.getState = () => state;
+
+    ({ router } = renderWithReduxRouter(
+      'imagewizard/b7193673-8dcc-4a5f-ac30-e9f4940d8346',
+      state
+    ));
+  };
+
+  test('selected repositories stored in and retrieved from form state', async () => {
+    await setUp();
+
+    screen.getByRole('heading', { name: /review/i });
+    await screen.findByText('Previously added custom repository unavailable');
```

Comment:
Awesome! Could be also nice to have the opposite test where you don't have an expired custom repo. That way we are protected from future misschanges.